article/app/controllers/ArticleController.scala

```diff
@@ -70,10 +74,10 @@ class ArticleController extends Controller with RendersItemResponse with Logging
 
   private def blockText(page: PageWithStoryPackage, number: Int)(implicit request: RequestHeader): Result = page match {
     case LiveBlogPage(liveBlog, _, _) =>
-      val blocks = liveBlog.blocks.collect {
+      val blocks = liveBlog.blocks.toSeq.flatMap{_.requestedBodyBlocks.get(Canonical.firstPage).toSeq.flatMap(_.collect {
```

Comment:
A lot of things happen on this line. Would it make sense to break it down in multiple variables (or multiple lines with comment) so it is more self-explanatory